repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/publication-system/acceptance-tests/publication-with-datasets/content.yaml

```diff
@@ -12,6 +12,7 @@
     hippostdpubwf:creationDate: 2017-11-03T13:35:36.812Z
     hippostdpubwf:lastModificationDate: 2017-11-03T13:35:36.813Z
     hippostdpubwf:lastModifiedBy: admin
+    hippotaxonomy:keys: ['people,-patients-and-georgraphy']
```

Comment:
not sure about that format :D I think it should be `[people, patients-and-georgraphy]`

Reply:
This is the key as defined in publication_taxonomy.yaml:

hippotaxonomy:key: people,-patients-and-georgraphy
...
hippotaxonomy:name: People, patients and georgraphy

Does this need changing? Not sure if you can have a key with whitespace?